These messages are trivial to trigger when running stress tests
like isic, and add no real value afaict.  Make them go away
instead of filling up logs pointlessly.

Signed-off-by: Dave Jones <davej@redhat.com>

--- linux-2.6.23.noarch/net/ipv4/netfilter/nf_conntrack_l3proto_ipv4.c~	2007-10-16 13:56:02.000000000 -0400
+++ linux-2.6.23.noarch/net/ipv4/netfilter/nf_conntrack_l3proto_ipv4.c	2007-10-16 13:56:08.000000000 -0400
@@ -176,8 +176,6 @@ static unsigned int ipv4_conntrack_local
 	/* root is playing with raw sockets. */
 	if (skb->len < sizeof(struct iphdr) ||
 	    ip_hdrlen(skb) < sizeof(struct iphdr)) {
-		if (net_ratelimit())
-			printk("ipt_hook: happy cracking.\n");
 		return NF_ACCEPT;
 	}
 	return nf_conntrack_in(PF_INET, hooknum, skb);
